applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -282,6 +282,9 @@ namespace Kratos
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, EXPONENTIAL_SATURATION_YIELD_STRESS)
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, ACCUMULATED_PLASTIC_STRAIN)
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, Vector, HIGH_CYCLE_FATIGUE_COEFFICIENTS)
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, EULER_ANGLE_PHI)
```

Comment:
since this is growing more and more can you please refactor the CLaw variables into a separate file?
Consider that this file is included in almost every other file in StructurallMechanics, adding more and more to this will make everything heavier

Plus it will be easier this way to move the CLaws to a separate application at some point